README.md

```diff
@@ -85,43 +84,46 @@ dra-example-driver-kubeletplugin-qwmbl           1/1     Running   0          1m
 ```
 
 And show the initial state of available GPU devices on the worker node:
-```console
-$ kubectl describe -n dra-example-driver nas/dra-example-driver-cluster-worker
-...
-Spec:
-  Allocatable Devices:
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-9ede7e32-5825-a11b-fa3d-bab6d47e0243
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-e7b42cb1-4fd8-91b2-bc77-352a0c1f5747
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-f11773a1-5bfb-e48b-3d98-1beb5baaf08e
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-0159f35e-99ee-b2b5-74f1-9d18df3f22ac
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-657bd2e7-f5c2-a7f2-fbaa-0d1cdc32f81b
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-18db0e85-99e9-c746-8531-ffeb86328b39
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-93d37703-997c-c46f-a531-755e3e0dc2ac
-    Gpu:
-      Product Name:  LATEST-GPU-MODEL
-      Uuid:          GPU-ee3e4b55-fcda-44b8-0605-64b7a9967744
-...
+```
+$ kubectl get resourceslice -o yaml
+apiVersion: v1
+items:
+- apiVersion: resource.k8s.io/v1alpha2
+  driverName: gpu.resource.example.com
+  kind: ResourceSlice
+  metadata:
+    creationTimestamp: "2024-04-17T13:45:44Z"
+    generateName: dra-example-driver-cluster-worker-gpu.resource.example.com-
+    name: dra-example-driver-cluster-worker-gpu.resource.example.comxktph
```

Comment:
Would it make sense to separate random suffix from the name, e.g.
`dra-example-driver-cluster-worker-gpu.resource.example.comxktph` -> `dra-example-driver-cluster-worker-gpu.resource.example.com.xktph` ?

Reply:
That is a question for @pohly  -- this comes from his library for generating the name

Reply:
The `GenerateName: dra-example-driver-cluster-worker-gpu.resource.example.com-` already has a trailing `-`. But as that prefix is long, Kubernetes truncates it and drops the trailing `-`.

One way to avoid that would be to truncate the `<node>-<driver>-` string in advance if it is too long, for example by cutting out some characters in the middle of each sub-component. Would that be better?

Reply:
> this comes from his library for generating the name

Just to be clear: that library produces `generateName`, not the `name` - that is the name generated by Kubernetes.